lib/static/css/main.css

```diff
@@ -2050,7 +2050,7 @@ input::placeholder {
   padding-bottom: 5px;
 }
 
-.fbLike {
+.fb-like {
```

Comment:
Could you add a comment here that we shouldn't change this class because it's determined by FB SDK?

Reply:
I think it should be in the js ? the css here just add extra style

Reply:
Yes but I renamed it thinking it was our own styling 😥

Reply:
Ah sure, let's write it in the JS then.